tidb-cloud/vector-search-data-types.md

```diff
@@ -18,9 +18,9 @@ Using vector data types provides the following advantages over using the [`JSON`
 - Dimension enforcement: You can specify a dimension to forbid inserting vectors with different dimensions.
 - Optimized storage format: Vector data types are optimized for handling vector data, offering better space efficiency and performance compared to `JSON` types.
 
-> **Note:**
+> **Note**
 >
-> Vector data types are only available for [TiDB Cloud Serverless](/tidb-cloud/select-cluster-tier.md#tidb-cloud-serverless) clusters.
+> TiDB Vector Search is only available for TiDB (>= v8.4) and [TiDB Cloud Serverless](/tidb-cloud/select-cluster-tier.md#tidb-cloud-serverless). It is not available for [TiDB Cloud Dedicated](/tidb-cloud/select-cluster-tier.md#tidb-cloud-dedicated).
```

Comment:
```suggestion
> TiDB Vector Search is only available for TiDB Self-Managed (>= v8.4.0) and [TiDB Cloud Serverless](/tidb-cloud/select-cluster-tier.md#tidb-cloud-serverless). It is not available for [TiDB Cloud Dedicated](/tidb-cloud/select-cluster-tier.md#tidb-cloud-dedicated).
```

Reply:
I'm not sure whether "TiDB Self-Managed" will bring negative impact on users' cognition. "TiDB Self-Managed" is a business oriented word, highlighting that user can self manage the database. In this context, user has no idea whether it is paid or not, and actually "self-manage" emphisized the price user may cost. I would rather like something like "TiDB Open-Source v8.4" or "TiDB Community v8.4" which highlights the "free of use" part. What do you think?

Reply:
This wording need to be carefully selected, as it could be one of the key that may cause users misunderstand TiDB for issues like https://github.com/pingcap/autoflow/issues/399

Reply:
The term “TiDB Self-Managed” is now consistently used across the TiDB documentation and PingCAP websites following the rebranding. Would it be better to use it here as well, rather than introducing a new concept?

![image](https://github.com/user-attachments/assets/6430e2dd-09aa-4bed-b805-e1e26c4e3d3f)

![image](https://github.com/user-attachments/assets/a2775fe5-9bc7-4602-9f90-20ab46983103)

Reply:
As the document indicated, "TiDB Self-Managed is a product option of TiDB". So it would be correct to be TiDB v8.4 instead of TiDB Self-Managed v8.4?

Reply:
When discussing TiDB in general, such as in SQL statement documentation, referring to TiDB v8.4 is appropriate. However, when explaining support information for different product options, explicitly distinguishing between TiDB Self-Managed and TiDB Cloud Serverless enhances clarity. For example:

![image](https://github.com/user-attachments/assets/3da78df0-ec44-4088-8dbc-719554f24304)

Reply:
I agree with @qiancai. The term “self-managed” in product names is generally a **neutral descriptor**, not directly indicative of cost. Instead, it refers to **how the product is managed**—by the user or customer, as opposed to being managed by the provider (e.g., in a fully-managed or cloud-hosted service).